### ISSUE TITLE
[Upgrade] Upgrade Laravel version to 5.8

### ISSUE THREAD

**Issue by claudiunicolaa** [code4romania/rvm-api#33]
Why Laravel version `5.5.x`? The framework is already at `5.8.x` version. Besides how the things work 5.5.x vs 5.8.x (isn't something very different, but exists a few little differences), the API extension of DateTime (`Carbon`) used into 5.5.x is deprecated.

```
*******************************************
 /!\ Warning, you're using a deprecated
 ¨¨¨ version of Carbon, we will soon stop
     providing support and update for 1.x
     versions, please upgrade to Carbon 2.
*******************************************

Most features of Carbon 1 are still available in Carbon 2.
See how to migrate: https://carbon.nesbot.com/docs/#api-carbon-2

Please consider upgrading your Laravel dependencies to be compatible with Carbon 2:
  - laravel/framework at least to version 5.8.0
```

**Comment by RaduCStefanescu** [comment]
Hi @claudiunicolaa 
Thank you for point this out. The project is still at scaffolding state, and much of the functionality is being freshly deployed. I will change the details of this issue to point to the upgrade of the laravel version to 5.8.

**Comment by RaduCStefanescu** [comment]
Fixed by #35
